keycloak-js/keycloak-js-tests.ts

```diff
@@ -0,0 +1 @@
+import { Keycloak } from 'keycloak-js';
```

Comment:
When I use the npm package like this:
```js
global.window = global;
console.log(require("keycloak"), require("keycloak").Keycloak);
```
I get `[Function: t] undefined`. So declaring a `Keycloak` export doesn't seem right.
Seems like [`export =`](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html) would be more accurate.
As always, test it in your own code.